lib/Controller/PageController.php

```diff
@@ -86,6 +105,15 @@ public function index(): TemplateResponse {
 			$this->userSession->getImpersonatingUserID() === null
 		);
 
+		$user = $this->userManager->get($this->uid);
+
+		$this->initialState->provideInitialState(
```

Comment:
There should be a unit test.

Reply:
I'll write some tests for the existing code. Let's implement the missing tests for the language code together.